types/stake.go

```diff
@@ -75,6 +75,8 @@ type ValidatorSet interface {
 	Slash(Context, crypto.PubKey, int64, int64, Dec)
 	Jail(Context, crypto.PubKey)   // jail a validator
 	Unjail(Context, crypto.PubKey) // unjail a validator
+
+	Delegation(Context, AccAddress, AccAddress) Delegation
```

Comment:
would be good to specify argument names here (for people who'll be looking at the interface later)

Reply:
hmm, we don't do that elsewhere - definitely a comment though